src/System.Private.CoreLib/shared/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -44,15 +50,10 @@ public SerializationInfo(Type type, IFormatterConverter converter)
             _names = new string[DefaultSize];
             _values = new object[DefaultSize];
             _types = new Type[DefaultSize];
+
             _nameToIndex = new Dictionary<string, int>();
-            _converter = converter;
-        }
 
-        [CLSCompliant(false)]
-        public SerializationInfo(Type type, IFormatterConverter converter, bool requireSameTokenInPartialTrust)
```

Comment:
I would use the CoreRT version here (forward from constructor with the extra argument to the constructor without it).

Reply:
Applied this change. Thanks